airflow/www/app.py

```diff
@@ -70,6 +73,31 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     app.json_encoder = AirflowJsonEncoder
 
     csrf.init_app(app)
+
+    def apply_middlewares(flask_app: Flask):
+        # Apply DispatcherMiddleware
+        base_url = urlparse(conf.get('webserver', 'base_url'))[2]
+        if not base_url or base_url == '/':
+            base_url = ""
+        if base_url:
+            flask_app.wsgi_app = DispatcherMiddleware(  # type: ignore
```

Comment:
This middleware is now optional. We use addresses in the form of `blocked` which do not work properly with this middleware. This middleware expects the addresses to be in the form `/blocked`. However, if we don't use this middleware, it doesn't matter.

Reply:
Should we remove it then?

Reply:
We don't use it in tests - test_views. This is still needed in production for some users.